File: bubblemania/Assets/BubbleMania_scripts/FinishedBubble.cs
using UnityEngine;
using System.Collections;

public class FinishedBubble : MonoBehaviour {

	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
	
	}
	void OnCollisionExit(Collision trk){
		Debug.Log ("hit232");
		if(trk.gameObject.tag=="bubble")
			Destroy(trk.gameObject);
		if(this.gameObject.tag=="projectile")
			Destroy(this.gameObject);
	}
}

File: bubblemania/Assets/BubbleMania_scripts/FinishedBubble.js
#pragma strict

function Start () {

}

function Update () {

}
function OnCollisionExit(trk : Collision){
	if(trk.gameObject.tag=="bubble"){
		Destroy(trk.gameObject);
		GameC.lives -= 1;
		if(GameC.lives<1)
			Application.LoadLevel("gameOver");
	}
}